return-404-page.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/webui/src/App.tsx b/webui/src/App.tsx
```

Comment:
I just had a quick look, but this should not be here.

Reply:
Oh totaly missed that one. Removed it.